### DISCUSSION TITLE
Não consigo validar o Radio-Button do Dynamic Form

### DISCUSSION THREAD

**Discussion by HorselessName** [po-ui/po-angular#1875] [Angular]
Olá, pessoal. Tudo bom?

Estou tentando validar o campo "Radio Button" gerado dentro do Dynamic Form, de acordo com essa documentação:
https://po-ui.io/documentation/po-dynamic-form

Eu fiz assim:
```HTML
<!-- Form Dinâmico: Permite Abrir a S.S. se os campos forem validos, com o Validate. -->
<po-dynamic-form
  #formAberturaOs
  [p-fields]="fieldsAberturaOs"
  [p-validate-fields]="this.fieldsValidadores"
  [p-validate]="onFormChange.bind(this)"
></po-dynamic-form>
```

```TypeScript
  // Validate Fields: Os campos que vão disparar o validate do form.
  fieldsValidadores: Array<string> = [
    'tipoDeProblema'];

  // Função ou serviço para validar as mudanças do formulário.
  onFormChange(changedValue: any): PoDynamicFormValidation {
    console.log("Campo alterado:", changedValue);
    return {}; // Retorna um objeto vazio para evitar erros
  }

  // Coleção que implementa a interface PoDynamicFormField.
  // Define e cria os campos dinamicamente.
  fieldsAberturaOs: PoDynamicFormField[] = [
    {
      divider: "Informações sobre o Problema",
      property: "tipoDeProblema",
      label: "Tipo de Problema",
      gridColumns: 9,
      required: true,
      showRequired: true,
      options: [
        {label: "Problema Mecânico", value: "M"},
        {label: "Problema Elétrico", value: "E"},
        {label: "Não sei", value: "O"},
      ],
      order: 1,
      formatModel: true,
    },

// Etc...
```

Mas, ali no meu log do Console, está aparecendo assim:

```JSON
{property: 'tipoDeProblema', value: undefined}
```

E gera o erro: `ERROR TypeError: Cannot convert undefined or null to object`

Alguém poderia me ajudar?

**Comment by anliben** [comment]
Boa Tarde @HorselessName , tudo bem?

Para podermos apoiá-lo precisaremos de mais detalhes para entender a situação

- Utilize o Stackblitz para reproduzir o problema: https://stackblitz.com/edit/po-ui

### Detalhe passo a passo a reprodução da ocorrência, inclua evidências utilizando imagens, vídeos ou gifs:

- Passo 1...*
- Passo 2...
- Detalhe o comportamento atual:
- Detalhe o comportamento que você espera ver:

### Informe a versão dos ambientes:
- PO UI: Ex. v15.8.0
- Angular: Ex. 15
- Node: Ex. 15

### Informe o Browser:
- [ ] Chrome
- [ ]  Firefox
- [ ] Edge
- [ ]  Safari
 - [ ] IE 11
- [ ] Outros

### Informe e o Sistema Operacional:
- [ ] Windows
- [ ] macOS
- [ ] Ubuntu
- [ ] Outros

### Aguardamos seu retorno!
